base_layer/core/src/chain_storage/db_transaction.rs

```diff
@@ -120,9 +120,17 @@ impl DbTransaction {
     }
 
     /// Adds a UTXO into the current transaction and update the TXO MMR.
-    pub fn insert_utxo(&mut self, utxo: TransactionOutput, header_hash: HashOutput, mmr_leaf_index: u32) -> &mut Self {
+    pub fn insert_utxo(
+        &mut self,
+        utxo: TransactionOutput,
+        header_hash: HashOutput,
+        header_height: u64,
```

Comment:
The header height here can be inferred on retrieval. There's no real need to store it